subprojects/logging/src/integTest/groovy/org/gradle/internal/logging/console/taskgrouping/BasicGroupedTaskLoggingFunctionalSpec.groovy

```diff
@@ -166,11 +167,13 @@ class BasicGroupedTaskLoggingFunctionalSpec extends AbstractConsoleGroupedTaskFu
         """
 
         when:
-        succeeds('run')
+        fails('run')
 
         then:
         result.groupedOutput.task(':a:log').outputs == ['Before', 'After']
+        result.groupedOutput.task(':a:log').outcome == 'FAILED'
         result.groupedOutput.task(':b:log').output == 'Interrupting output'
+        result.groupedOutput.task(':b:log').outcome == null
```

Comment:
Very minor. I don't know if we would rather return `SUCCESS` instead of `null`. This is [what TestKit does](https://docs.gradle.org/current/javadoc/org/gradle/testkit/runner/TaskOutcome.html#SUCCESS).

Reply:
Actually, the original value is `null`: https://github.com/gradle/gradle/blob/261e74483d65f4126b64fd369001fcbfe5fcaa54/subprojects/core/src/main/java/org/gradle/api/internal/tasks/TaskExecutionOutcome.java#L24